views/content/approval.html

```diff
@@ -12,22 +12,23 @@
 <h1 class="base--h1">{{title}}</h1>
 
 {% if revision %}
-  <h2 class="base--h2">{{config.actions.approve}} :: [TBAdded: Primary Key Contents]</h2>
+  <h2 class="base--h2">{{config.actions.approve|capitalize}} :: [TBAdded: Primary Key Contents]</h2>
 
   {% if data %}
+    <div class="listing">
     {% for attr, inputs in data %}
-      <h3>{{attr}}</h3>
+      <h3 class="listing--h3">{{attr|capitalize}}</h3>
 
       {% if inputs.length %}
         {# repeatable content #}
         {% for rpt in inputs %}
           {% for input, value in rpt %}
             {% if loop.first %}
-              <dl class="base--dl">
+              <dl class="listing--dl">
```

Comment:
I think this html might need some rethinking. (not the DL, DL's are cool)

This DL is what would be triggered on a repeatable. So, when repeated, the DL doesn't close until all content is gone through.

So, for something using multi-inputs, like the [quote plugin](https://github.com/punchcard-cms/input-plugin-quote), if not repeated it would print out:

```
H3 Quote
dl
  dt quote
  dd (content)
/dl
dl
  dt author
  dd (content)
/dl
dl
  dt source
  dd (content)
/dl
```

if it is repeated it would print out:

```
H3 Quote
dl
  dt quote
  dd (content)
  dt author
  dd (content)
  dt source
  dd (content)
/dl

dl
  dt quote
  dd (content)
  dt author
  dd (content)
  dt source
  dd (content)
/dl
```